### DISCUSSION TITLE
Git - dúvida (Tree/branch - visual)

### DISCUSSION THREAD

**Discussion by gabrieldarezzo** [frontendbr/forum#364] [Perguntas e Respostas]
Estava 'brincando' aqui no BitBucket e olhando a tree deles e tive uma ideia.
 
![image](https://cloud.githubusercontent.com/assets/4562368/20184249/2d906198-a74f-11e6-92a4-e09f96a3c58c.png)

Minha ideia seria exibir a tree de forma para auxiliar pequenas equipes de desenvolvimento exibindo em tempo real por exemplo tree/branch por usuário.

Existe alguma ferramenta nesse aspecto?
Já desenvolvi em alguns lugares onde ficava telões parados no departamento de T.I. e seria uma boa utilizar um desses para exibir quais os 'chamados' a galera está trabalhando.

Alguém já viu algo parecido?

**Comment by robsonbittencourt** [comment]
Não, mas achei muito interessante a ideia. Acompanhando.

**Comment by felipefialho** [comment]
@gabrieldarezzo O GitKraken apresentado aqui -> #265, não funciona mais ou menos assim?

**Comment by gabrieldarezzo** [comment]
Eu até olhei sua dica @LFeh 
To pensando em algo mais simples. ( e openSource)

Seria só uma exibição, estilo:

> git log --graph --full-history --all --pretty=format:"%h%x09%d%x20%s"

Acredito que utilizando a API do Git/BitBucket é possível fazer tranquilamente.

Assim q sobrar um tempo começo a me iterar mais no assunto.

Agradeço as respostas
